ee/modules/502-delivery/templates/argocd/repo-server/argocd-repo-server-deployment.yaml

```diff
@@ -78,7 +78,7 @@ spec:
         - image:  {{ include "helm_lib_module_image" (list . "werfArgocdCmpSidecar") }}
           imagePullPolicy: IfNotPresent
           name: werf-argocd-cmp-sidecar
-          {{- include "helm_lib_module_container_security_context_read_only_root_filesystem_capabilities_drop_all" . | nindent 10 }}
+          {{- include "helm_lib_module_container_security_context_capabilities_drop_all_and_run_as_user_custom" (list . 999 999) | nindent 10 }}
```

Comment:
Why we run on user 999 ? In all modules we use nobody user and have helper for it

Reply:
We have an issue with this sidecar. It should write to /home/argocd on behalf of user argocd, but it tries to write to /nobody and fails. Argocd user id is 999. I should leave a comment for this.

Reply:
As I can see, there are many discussions about changes in helm lib. I suggest this [pr](https://github.com/deckhouse/deckhouse/pull/3779) to resolve root cause.